modules/telco-core-scheduling.adoc

```diff
@@ -10,17 +10,28 @@ New in this release::
 * No reference design updates in this release
 
 Description::
-* The scheduler is a cluster-wide component responsible for selecting the right node for a given workload. It is a core part of the platform and does not require any specific configuration in the common deployment scenarios. However, there are few specific use cases described in the following section.
+The scheduler is a cluster-wide component responsible for selecting the right node for a given workload.
+It is a core part of the platform and does not require any specific configuration in the common deployment scenarios.
+However, there are few specific use cases described in the following section.
+
 NUMA-aware scheduling can be enabled through the NUMA Resources Operator.
```

Comment:
This content renders kind of oddly, in that starting with "NUMA-aware," the content is no longer lined up with the above paragraphs. See the description here: http://88622--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.html#telco-core-scheduling_telco-core